etc/kayobe/kolla.yml

```diff
@@ -224,7 +219,7 @@ stackhpc_ubuntu_jammy_repos:
   - "deb {{ stackhpc_repo_ubuntu_jammy_url }} jammy-updates main universe"
   - "deb {{ stackhpc_repo_ubuntu_jammy_url }} jammy-backports main universe"
   - "deb {{ stackhpc_repo_ubuntu_jammy_security_url }} jammy-security main universe"
-  - "deb {{ stackhpc_repo_ubuntu_cloud_archive_url }} jammy-updates/{{ openstack_release }} main"
+  - "deb {{ stackhpc_repo_ubuntu_cloud_archive_url }} jammy-updates/antelope main"
```

Comment:
I don't think this is too helpful. It shouldn't need to change between releases.

Reply:
This was actually causing issues because `{{ openstack_release }}` resolves to `2023.1` rather than `antelope`

Reply:
Doh. Let's introduce a new variable, openstack_release_name. It can be annotated to state that it will need to be updated.